README.md

```diff
@@ -1,4 +1,4 @@
-Courseography
+[Courseography](https://courseography.cdf.toronto.edu)
```

Comment:
We're trying to use `https://courseography.cs.toronto.edu` now (the `cdf` domain is deprecated, I think)